include/glow/Base/Tensor.h

```diff
@@ -551,6 +551,36 @@ class Tensor final {
   /// Dump a textual representation of the shape of this Tensor to std::string.
   std::string getShapeToString() const;
 
+  /// Dump the data content of the tensor to a raw-binary file without any
```

Comment:
As we are going to have many different serialization/deserialization methods, e.g. raw and more complex ones, and most of them do not really need to access the internal representation of Tensors, I'd suggest moving this functionality into dedicated TensorSerialization.h/.cpp files.

Reply:
Done.